Fix some command to make it works

Wr3nch-ren opened the pull request:
Fix adduser no be able to create munge user and slurm user.
Fix slurm's version to match with installed version in the workshop.

Wr3nch-ren committed:
Fix 02-MUNGE adduser command

Wr3nch-ren committed:
Change 04-SLURM command to match with current slurm version.

paleumm commented:
Nice catch on second commit!

On first commit, your solution is work but it's not quite what we intended to do

We want to create munge (and slurm) user to have these criteria
- no shell login
- no home directory
- system user

We can try this command instead

for munge user
```bash
sudo useradd --system -u "$MUNGEUSER" -g munge -d /var/lib/munge -s /sbin/nologin -c "MUNGE" --no-create-home munge
```

for slurm user
```bash
sudo useradd --system -u "$SLURMUSER " -g slurm -d /var/lib/slurm -s /sbin/nologin -c "SLURM" --no-create-home slurm
```

The original error is caused by adding params in wrong order and too many params

If you want you can test and add new commit with the new commands

Wr3nch-ren committed:
Fix 02-MUNGE README.md to match with suggested criteria

Munge and Slurm user is now created with these criteria:

no shell login
no home directory
system user

as paleumm's comment

Wr3nch-ren committed:
Merge branch 'HPC-Thailand:main' into main

Wr3nch-ren commented:
Ok, I changed the command to match with your suggestion, thank you for your clarification.
